frb_codegen/src/commands.rs

```diff
@@ -301,6 +364,58 @@ fn guess_toolchain(dart_root: &str) -> anyhow::Result<DartToolchain> {
     Ok(DartToolchain::Dart)
 }
 
+fn ensure_dependencies(dart_root: &str, key: &str) -> anyhow::Result<Option<VersionReq>> {
+    debug!(
+        "Checking presence of {} in dependencies in {}",
+        key, dart_root
+    );
+    let manifest_file = read_file(dart_root, "pubspec.yaml")?;
+    info!("{:#?}", manifest_file);
```

Comment:
uber-nit: remove debugging logs when ready to merge

Or, we can make it `debug!` or `trace!`

Reply:
yes I'm gonna clean all these up,
sorry I'm a bit forgetful there's all these `info!` scattered here and there 😅

Reply:
No worries :)

Reply:
it should be good too.

Reply:
Feel free to tap "resolve conversation" when done :)